io.github.TheWisker.Cavasik.json

```diff
@@ -0,0 +1,83 @@
+{
+    "app-id" : "io.github.TheWisker.Cavasik.desktop",
```

Comment:
there is never `.desktop` in the app-id.

```suggestion
    "app-id" : "io.github.TheWisker.Cavasik",
```

Reply:
Thanks. Fixed it!